daft/table/table.py

```diff
@@ -524,6 +524,7 @@ def read_parquet_into_pyarrow(
     io_config: IOConfig | None = None,
     multithreaded_io: bool | None = None,
     coerce_int96_timestamp_unit: TimeUnit = TimeUnit.ns(),
+    string_encoding: str | None = "utf-8",
```

Comment:
We should probably do `string_encoding: Literal["utf-8"] | Literal["raw"] = "utf-8"`

Reply:
I like this change. Will have them up asap.

Reply:
Updated!